src/main/java/gregtech/api/recipes/Recipe.java

```diff
@@ -60,6 +62,15 @@ public static String formatChanceValue(int outputChance) {
 
     private final RecipePropertyStorage recipePropertyStorage;
 
+    private static final ItemStackHashStrategy hashStrategy = ItemStackHashStrategy.builder()
+            .compareItem(true)
+            .compareCount(true)
+            .compareDamage(true)
+            .compareTag(true)
+            .build();
```

Comment:
Nitpick but I would rather see a default hash strategy in `ItemStackHashStrategy` just called `comparingAll()` to make this nicer, since that seems like a pretty normal case